gemrb/core/GameScript/GSUtils.cpp

```diff
@@ -2944,7 +2944,7 @@ void AddXPCore(const Action *parameters, bool divide)
 		// give xp to everyone
 		core->GetGame()->ShareXP(atoi(xpvalue), 0);
 	}
-	core->PlaySound(DS_GOTXP, SFX_CHAN_ACTIONS);
+	core->PlaySound(DS_GOTXP, SFX_CHAN_GUI);
```

Comment:
Did you figure these out with testing via sndchann,2da? It's quite unintuitive that actions wouldn't use the actions channel.

Reply:
Hm, no. I thought of things that happen on the map (like lock picking etc.) vs. things help you notice something on the GUI for information.

Reply:
I suggest you test it in the original then. Set the various channels to 0 and you should be able to confirm which one is used. Actions can be ran through the cheat console too.

Reply:
I think this can be reverted then with the other idea of looking at the relative flag. The idea was to keep some noise out of the reverb case spectrum.